lib/Twig/Extension/Core.php

```diff
@@ -852,6 +852,30 @@ function twig_in_filter($value, $compare)
     return false;
 }
 
+/** 
+ * Returns a trimmed string.
+ * 
+ * @param string $string
+ * @param string $character_mask
+ * @param string $mode
+ * @return string
+ */
+function twig_trim_filter($string, $character_mask = null, $type = 'both')
```

Comment:
@julienfalque would that work for named parameters? As the documentation has "character_mask".